src/MvvmNano.Forms/MvvmNanoApplication.cs

```diff
@@ -7,6 +8,26 @@ namespace MvvmNano.Forms
     /// </summary>
     public class MvvmNanoApplication : Application
     {
+
+        /// <summary>
+        /// A collection of MasterDetailData that represents available detail pages.
+        /// </summary>
+        public ObservableCollection<MasterDetailData> MasterDetails { get; set; } = new ObservableCollection<MasterDetailData>();
```

Comment:
I don't see those properties in the `MvvmNanoApplication` yet. Maybe one could introduce another, more specific, Application class, which inherits from this one and extends it for all those master detail stuff.